compiler/AST/symbol.cpp

```diff
@@ -2216,10 +2211,16 @@ const char* toString(FnSymbol* fn) {
     if (skipParens == false) {
       retval = astr(retval, ")");
     }
+  }
 
-    if (developer  == true) {
-      retval = astr(retval, " [", istr(fn->id), "]");
-    }
+  if (fn->where && fn->where->body.length == 1) {
```

Comment:
Could you clarify the purpose of the `fn->where->body.length == 1` condition to me?

Reply:
In the second commit message I mentioned it, but misspelled the word "shouldn't" as "should".

The compiler sometimes inserts multi-statement where clauses and we shouldn't print them.  One such case is for functions that have a queried argument.